packages/api/test/scripts/worker-globals.js

```diff
@@ -11,3 +11,4 @@ export const VERSION = 'test'
 export const COMMITHASH = 'test'
 export const MAINTENANCE_MODE = 'rw'
 export const METAPLEX_AUTH_TOKEN = 'metaplex-test-token'
+export const LOGTAIL_TOKEN = ''
```

Comment:
Please add the docs on how this is configured for production/dev

Reply:
fixed